fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisParser.g4

```diff
@@ -428,7 +428,11 @@ selectClause
 
 selectColumnClause
     : namedExpressionSeq
-    | ASTERISK EXCEPT LEFT_PAREN namedExpressionSeq RIGHT_PAREN
+    | ASTERISK exceptOrReplace+
```

Comment:
if except and replace could not use togather, why not use ? instead of +

Reply:
they can use together, you can see this feature in the regression test